spec/index.bs

```diff
@@ -2497,13 +2497,9 @@ enum GPUPowerPreference {
 
         <dl dfn-type=dfn dfn-for="feature level string">
             : <dfn noexport>"core"</dfn>
-            :: No effect.
+            :: Requests a Core Level Adapter. If the implementation does not support Core Level, a null Adapter will be returned.
             : <dfn noexport>"compatibility"</dfn>
-            :: No effect.
-
-                Note:
-                This value is reserved for future use as a way to opt into additional validation restrictions.
-                Applications should not use this value at this time.
+            :: Requests a Compatibility Adapter. If the implementation supports Compatibility mode, the returned Adapter will support Compatiblity mode. All Devices created from it will enforce Compatiblity mode validation. If the implementation does not support compatibility mode, a Core level Adapter will be returned, with the "core-features-and-limits" Feature enabled.
```

Comment:
> All Devices created from it will enforce Compatiblity mode validation.

Only by default. Might want to introduce the "Core-defaulting"/"Compatibility-defaulting" language.

Also "core-features-and-limits" may be available on the adapter even if it's Compatibility-defaulting.

Reply:
Done. I've basically merged the above text with the definitions from the proposal.